src/python/crash_analysis/stack_parsing/stack_symbolizer.py

```diff
@@ -45,6 +45,7 @@
 from platforms.android import adb
 from platforms.android import fetch_artifact
 from platforms.android import settings
+from platforms.android import symbols_downloader
```

Comment:
I think you forgot to add this file?

Reply:
+1, need to review this.
also, symbols_downloader as android_symbols_downloader to be more explicit.